_posts/2021-04-27-Optimising-ux-for-software.markdown

```diff
@@ -0,0 +1,76 @@
+<br /><br />
+
+## Shared Understanding
+
+A lack of shared understanding about software constraints, and therefore the system in which a user experience operates in, means decisions are made with partial context; impeding this flow and introducing a slow latency. Other than designers and developers working together and feedback loops, another aspect is providing visibility of the current system’s constraints. Value Stream Mapping (VSM) is an approach to do that.
+
+VSM is a method used to map value from idea to customer; looking at the processes, pain points, latency and actors involved that may impede or support the flow of value through a system. It is similar to a user journey map except the focus is on software. Read more about VSM from [Patrick Anderson](https://blog.tasktop.com/did-your-organization-wake-up-as-a-software-company-consider-value-stream-management-vsm/).
+
+There is value in designers being part of a VSM workshop. They have greater context around how the current system operates, the constraints, gaps and opportunities and how that is feeding into poor user experience. For example, with a recent client looking at AI/ML, we did not have a complete understanding of how the AI/ML worked, which meant that when we created new screens we made assumptions about the flow. It was not until we re-engaged the client that we had more context. This lack of knowledge was a constraint; it slowed us down. Now VSM is not integral to DevOps, however, VSM provides visibility into the constraints and gaps within a system and this visibility is integral to the value of DevOps. Mapping value from idea to customer provides a designer with more context on the current system and an idea of future state. This context coupled with user research leads to data driven decisions.
+<br /><br />
+
+## Why UX and DevOps
+
+DevOps without UX is possible however your return on investment and delivery success is impeded and we see this with website latency costing $2.6 billion ([Design Resources ](https://techjury.net/blog/user-experience-stats/#gref)). I would not recommend it. It is all well and good to accelerate technology delivery but that value is lost if there is no clear strategy and no understanding of users.
```

Comment:
> DevOps without UX is possible however your return on investment and delivery success is impeded and we see this with website latency costing $2.6 billion ([Design Resources ](https://techjury.net/blog/user-experience-stats/#gref)).

This statistic seems a bit dubious.  
Following the sources took me here: https://econsultancy.com/slow-loading-websites-cost-retailers-1-73bn-in-lost-sales-each-year/
But I can't find any reference to the original 'research'.

Prefer quoting the original sources where you can.  Also what currency?  There's been a conversion between the 'original' source and the figure you quote.

The original statistic related to slow loading times - do you need to make the link to UX, and why this is not just a prioritisation or engineering problem?